dask/dataframe/core.py

```diff
@@ -1755,6 +1757,21 @@ def astype(self, dtype):
         empty = self._pd.astype(dtype)
         return map_partitions(pd.DataFrame.astype, empty, self, dtype=dtype)
 
+    def info(self):
+        """
+        Concise summary of a Dask DataFrame.
+        """
+        lines = list()
+        lines.append(str(type(self)))
+        lines.append('Data columns (total %d columns):' % len(self.columns))
+        dtypes = self.dtypes
+        template = "%s    %s"
```

Comment:
These won't align if the column names aren't the same length. Not a big deal, but the equivalence test used below will only be correct if they are. Up to you if you want to fix or not.